media-libs/libsdl2/libsdl2-2.28.5.ebuild

```diff
@@ -110,6 +111,7 @@ src_prepare() {
 	# Unbundle some headers.
 	rm -r src/video/khronos || die
 	ln -s "${ESYSROOT}/usr/include" src/video/khronos || die
+	use vulkan || sed -i '/testvulkan$(EXE) \\/d' "test/Makefile.in" || die
```

Comment:
Please don't do these `||` chains, they are horribly unreadable and prone to mistakes in the future.

Reply:
Done